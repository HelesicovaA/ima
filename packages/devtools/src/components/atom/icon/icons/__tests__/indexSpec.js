import Icons from '../';

describe('icons', () => {
  it('should match snapshot', () => {
    expect(Icons).toMatchSnapshot();
  });
});
