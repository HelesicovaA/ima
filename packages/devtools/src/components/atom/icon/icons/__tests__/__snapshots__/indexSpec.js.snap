// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons should match snapshot 1`] = `
Object {
  "arrowDown": [Function],
  "arrowUp": [Function],
  "check": [Function],
  "close": [Function],
  "cog": [Function],
  "copy": [Function],
  "edit": [Function],
  "enable": [Function],
  "more": [Function],
  "reset": [Function],
  "search": [Function],
  "trash": [Function],
}
`;
